components/Questionnaire/Confirmation/index.vue

```diff
@@ -261,6 +265,26 @@ export default {
         })
       }
       return villages
+    },
+    isFormCompleted () {
+      return !!((
```

Comment:
what is `!!` used for, I'm a little confused.

Reply:
It is used to force the conversion of any value to the corresponding boolean primitive (true / false).

reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT

Reply:
wow thanks 🙌🏻